Kitodo/src/test/java/org/kitodo/config/xml/fileformats/FileFormatsConfigIT.java

```diff
@@ -0,0 +1,46 @@
+package org.kitodo.config.xml.fileformats;
+
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Dependency of `org.hamcrest` is not defined in core module of Kitodo. If you want to use it, add this library to the `Kitodo/pom.xml` for scope `test`.

Reply:
Fixed